src/bin/readstat.c

```diff
@@ -78,6 +48,9 @@ const char *format_name(int format) {
 
     if (format == RS_FORMAT_SAS_CATALOG)
         return "SAS catalog file (SAS7BCAT)";
+    
+    if (format == RS_FORMAT_CSV)
+        return "CSV (NSD)";
```

Comment:
That would be the Norwegian abbreviation of Norwegian Centre for Research Data. I've changed it to simply say "CSV" now.